GitCommands/RevisionReader.cs

```diff
@@ -131,33 +135,19 @@ public sealed class RevisionReader : IDisposable
                 {
                     token.ThrowIfCancellationRequested();
 
-                    if (TryParseRevision(module, chunk, stringPool, logOutputEncoding, out var revision))
+                    if (TryParseRevision(module, chunk, logOutputEncoding, sixMonths, hasPathFilter, out var revision))
                     {
-                        if (revisionPredicate is null || revisionPredicate(revision))
```

Comment:
`revisionPredicate` is not used / applied.

Reply:
reapplied